fastlane/metadata/en-US/app_store_screenshot_6.txt

```diff
@@ -0,0 +1 @@
+Protect your notes with a psscode lock
```

Comment:
Looks like there's a typo here
```suggestion
Protect your notes with a passcode lock
```

Reply:
Wooops. I did fix this in another branch, but I'll cherry pick it here 👍